src/Services.Remoting/Client/OmexServiceProxyFactory.cs

```diff
@@ -34,7 +42,8 @@ public static void WithTransportSettings(FabricTransportRemotingSettings transpo
 		/// </summary>
 		/// <remarks>
 		/// The default implementation uses insecure connections.
-		/// To use secure connections call <see cref="WithTransportSettings"/> prior to calling .Instance.
+		/// To use secure connections either add a TransportSettings section to the service manifest
+		/// or call <see cref="WithCustomTransportSettings"/> prior to calling .Instance to use custom transport settings configuration.
```

Comment:
nit: I assumed you wanted to refer to calling `.Instance`
```suggestion
		/// or call <see cref="WithCustomTransportSettings"/> prior to calling `.Instance` to use custom transport settings configuration.
```